docs/data-sources/modelarts_dataset_versions.md

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "AI Development Platform (ModelArts)"
+---
+
+# huaweicloud_modelarts_dataset_versions
+
+Use this data source to get a list of ModelArts dataset versions.
+
+## Example Usage
+
+```hcl
+variable "dataset_id" {}
+data "huaweicloud_modelarts_dataset_versions" "test" {
+  dataset_id = var.dataset_id
+}
+```
+
+## Argument Reference
+
+The arguments of this data source act as filters for querying the available dataset versions in the current region.
+ All dataset versions that meet the filter criteria will be exported as attributes.
+
+* `region` - (Optional, String) Specifies the region in which to obtain dataset versions. If omitted, the
+ provider-level region will be used.
+
+* `dataset_id` - (Required, String) Specifies the ID of dataset.
+
+* `split_ratio` - (Optional, String) Specifies the range of splitting ratio which randomly divides a labeled sample
+ into a training set and a validation set. Separate the minimum and maximum split ratios with commas,
+ for example: "0.0,1.0".
+
+* `name` - (Optional, String) Specifies the name of the dataset version.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - Indicates a data source ID.
+
+* `versions` - Indicates a list of all dataset versions found. Structure is documented below.
+
+The `versions` block contains:
+
+* `version_id` - The ID of the dataset version.
```

Comment:
just name this to `id` is ok, as it's already under versions block

Reply:
fix